spec/takeaway_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu: menu, order_klass: order_klass, checkout: checkout) }
```

Comment:
Metrics/LineLength: Line is too long. [102/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)